content/blog/2021/cfp-announcement.md

```diff
@@ -0,0 +1,99 @@
+Title: Call for Proposals for PyCon India 2021 is now open!
+Date: 2021-03-04 17:00
+Author: Nikhil Maan (@Sc0rpi0n101)
+Category: 2021
+Slug: cfp-announcement

```

Comment:
`cfp-announcement-2021` ?

This might create conflicts later on.

Reply:
2021 is prefixed to the path always.